### ISSUE TITLE
Running the Evaluation Job is not working?

### ISSUE THREAD

**Issue by adarvit** [tensorflow/models#3936]
hey all, i am using the code from g3doc for running locally in order to do evaluation job

but i am getting the following warning and then the cmd is seems to be frozen:

![capture](https://user-images.githubusercontent.com/33596858/38556661-1a77b78a-3cd3-11e8-9450-a6b39aa04949.PNG)

is it ok? am i missing something?

**Comment by FalcoGer** [comment]
for me eval is a bit slow. just give it some time to run.
in your pipeline config check that it doesn't evaluate too many pictures if your validation set is too big.
eval runs your trained model against a validation set, that should be different from your training set. thus it applies the network to those images. the time increases with more images and a more complex network.

**Comment by adarvit** [comment]
thanks for the reply. I am little bit confused
 on the config file: eval_input_reader is not for the test set? if that refers to validation set, so how i can evaluate on my test set?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
